src/prefect/orion/utilities/schemas.py

```diff
@@ -157,6 +157,10 @@ class Config:
         else:
             extra = "ignore"
 
+        json_encoders = {
+            SecretField: lambda v: v.dict() if getattr(v, "dict", None) else str(v)
+        }
```

Comment:
This feels... weird. Can we handle `SecretDict` separately?

Reply:
I did this to avoid a circular import. Does it make sense to include `SecretDict` in this file?

Reply:
Ah yeesh. Just note it in a comment and we can tackle it whenever I declare war on circular imports.